env/src/main/java/com/sap/cloud/security/config/Environments.java

```diff
@@ -76,8 +79,8 @@ public static Environment readFromInput(InputStream input) {
 	}
 
 	private static boolean isK8sEnv() {
```

Comment:
idea I had yesterday, you can also move that to K8sEnvironment (in case you need that for testability)